gapis/trace/android/profile/slices.go

```diff
@@ -208,75 +196,3 @@ func (e *extras) get(ctx context.Context, argSet int64) []*service.ProfilingData
 	}
 	return extras
 }
-
-type groupTreeNode struct {
-	id   int32
-	name string
-	link sync.SubCmdRange
-
-	children []groupTreeNode // sorted by child.link
-}
-
-type groupTree struct {
-	nextID int32
```

Comment:
nextID? What does next refer to in this case?

Reply:
Added a comment.